pkg/engine/doc.go

```diff
@@ -18,26 +18,26 @@
 //
 // Let's get operation Preview as an example to demonstrate how the three parts cooperate in an actual operation.
 //
-//                        +-------------+
-//                        |  Operation  |
-//                        |  Preview    |
-//                        +-------------+
-//                               |
-//                               |
-//                        +-------------+
-//                        |  Operation  |
-//              +---------|  Engine     |----------+
-//              |         +-------------+          |
-//              |                                  |
-//       +-------------+                    +-------------+
-//       |   State     |                    |   Runtime   |
-//       +-------------+                    +-------------+
+//	                 +-------------+
```

Comment:
It is recognized as blocker format issue in golangci-lint v1.50.1.
I upgrade the golangci-lint version because of the old version do not support go 1.18.